### PR TITLE
adding new nursery rules

### PR DESCRIPTION
**Pull request by mike-hunhoff**

### CODE REVIEW

File: nursery/execute-shell-command-via-windows-remote-management.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  meta:
+    name: execute shell command via Windows Remote Management
+    namespace: c2/shell
+    author: michael.hunhoff@fireeye.com
+    scope: function
+  features:
+    - and:
+      - or:
+        - api: wsmsvc.WSManRunShellCommand
+        - api: wsmsvc.WSManRunShellCommandEx
```
Comment (williballenthin) [inline nursery/execute-shell-command-via-windows-remote-management.yml:12]: whoa i had no idea it was this easy

File: nursery/execute-shell-command-via-windows-remote-management.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# generated using capa explorer for IDA Pro
+rule:
+  meta:
+    name: execute shell command via Windows Remote Management
+    namespace: c2/shell
```
Comment (williballenthin) [inline nursery/execute-shell-command-via-windows-remote-management.yml:5]:
```suggestion
    namespace: host-interaction/process/create
```

?

File: nursery/spoof-parent-pid.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  meta:
+    name: spoof parent PID
+    namespace: anti-analysis/anti-forensic
+    author: michael.hunhoff@fireeye.com
+    scope: basic block
+    references: https://blog.f-secure.com/detecting-parent-pid-spoofing/
+  features:
+    - and:
+      - api: kernel32.UpdateProcThreadAttribute
+      - number: 0x20000 = PROC_THREAD_ATTRIBUTE_PARENT_PROCESS
```
Comment (williballenthin) [inline nursery/spoof-parent-pid.yml:12]: this one is a great technique

**Review by mr-tz — APPROVED** [review]
great additions!

**Pull request merged**